examples/open-meanstack/open-meanstack.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.1-dev
+
+services:
+- name: mongo
+  containers:
+  - image: mongo
+    ports:
+    - port: 27017:27017
+      type: external
+    mounts:
+    - volumeName: mongo-persistent-storage
+      mountPath: /data/db
+  emptyDirVolumes: 
+    - name: mongo-persistent-storage
```

Comment:
if you wanna use the volumes why not use the root level volume directive? Because using this we will lose the data!